keps/sig-storage/0661-statefulset-volume-resize/kep.yaml

```diff
@@ -0,0 +1,56 @@
+title: StatefulSet volume resize
+kep-number: 0661
+authors:
+  - "@shalini-b"
+owning-sig: sig-apps
+participating-sigs:
+  - sig-storage
+status: provisional
+creation-date: 2021-07-13
+reviewers:
+  - "@kow3ns"
+  - "@gnufied"
+  - "@msau42"
+  - "@xing-yang"
+approvers:
+   - "@kow3ns"
+   - "@xing-yang"
+
```

Comment:
Add a reference to the previous KEP:

see-also:
  - "https://github.com/kubernetes/enhancements/pull/1848/"